storage/storage.hpp

```diff
@@ -71,12 +76,20 @@ struct NodeAttrs
   string m_nodeLocalName;
 
   /// Node id and local name of the parents of the node.
-  /// For the root m_parentInfo.m_id == "" and m_parentInfo.m_localName == "".
+  /// For the root |m_parentInfo| is empty.
   /// Locale language is a language set by Storage::SetLocale().
-  /// Note. Most number of nodes have only one parent. But in case of a disputed territories
+  /// \note Most number of nodes have only one parent. But in case of a disputed territories
   /// an mwm could have two or even more parents. See Country description for details.
   vector<CountryIdAndName> m_parentInfo;
 
+  /// Node id and local name of the first level parents (root children nodes)
+  /// if the node has first level parent(s). Otherwise |m_topmostParentInfo| is empty.
+  /// That means for the root and for the root children |m_topmostParentInfo| is empty.
+  /// Locale language is a language set by Storage::SetLocale().
```

Comment:
CountryIdAndName has a member with locale name. Probably it's worth writing which Locale is used here.